src/test/java/bio/terra/service/filedata/azure/tables/TableDirectoryDaoConnectedTest.java

```diff
@@ -47,10 +50,12 @@
 public class TableDirectoryDaoConnectedTest {
   private static final Logger logger =
       LoggerFactory.getLogger(TableDirectoryDaoConnectedTest.class);
+  private Dataset dataset;
   private UUID datasetId;
+  private UUID snapshotId;
   private TableServiceClient tableServiceClient;
   private List<String> directoryEntriesToCleanup = new ArrayList<>();
-  private String tableName;
+  private String snapshotTableName;
 
   @Autowired AzureSynapsePdao azureSynapsePdao;
```

Comment:
Ahh the multi-line suggestion isn't working for me here, but I think we only need these components:
```
  @Autowired ConnectedOperations connectedOperations;
  @Autowired private ConnectedTestConfiguration testConfig;
  @MockBean private IamProviderInterface samService;
  @Autowired TableDirectoryDao tableDirectoryDao;
  @Autowired AzureUtils azureUtils;
```

Reply:
Thanks for catching these! I'm used to things that aren't used getting greyed out in intellij, but that doesn't seem to be the case 😞